.github/workflows/pubsys-exercise.yaml

```diff
@@ -0,0 +1,27 @@
+name: pubsys
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - 'tools/**'
+jobs:
+  check:
+    runs-on: [self-hosted, linux, x64]
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup toolchain install 1.53.0 && rustup default 1.53.0
+      - run: cargo install --version 0.30.0 cargo-make
+      - run: cargo make -e BUILDSYS_JOBS=12
+      - run: cargo make repo
+      - run: mv build/repos/default build/repos/test
```

Comment:
What's this step for?

Reply:
Refresh-repo will fail when refreshing the locally built repo since it'll refuse to update it in place (by design). So the built repo must be moved out to a different location before trying to refresh it.